.github/workflows/hourly-run.yml

```diff
@@ -0,0 +1,105 @@
+on:
+  schedule:
+    - cron:  '30 * * * *' # every hour at `x:30`
+  workflow_dispatch:
+
+name: Hourly check
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  self-care:
+    name: Flake self-check
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+      - name: Check Nix flake inputs
+        uses: DeterminateSystems/flake-checker-action@v5
+        with:
+          fail-mode: true
+          nixpkgs-keys: nixpkgs
+
+  test:
+    if: github.repository == 'fedimint/fedimint'
+    strategy:
+      matrix:
+        host:
+          - linux
+        include:
+          - host: linux
+            runs-on: buildjet-8vcpu-ubuntu-2004
+            timeout: 75
+            run-tests: true
+
+    name: "Test on ${{ matrix.host }}"
+    runs-on: ${{ matrix.runs-on }}
+    timeout-minutes: ${{ matrix.timeout }}
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: cachix/install-nix-action@v26
+        with:
+          nix_path: nixpkgs=channel:nixos-23.11
+          extra_nix_config: |
+            connect-timeout = 15
+            stalled-download-timeout = 15
+
+      - uses: cachix/cachix-action@v14
+        with:
+          name: fedimint
+          authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
+        continue-on-error: true
+
+      - name: Tests (5 times more)
+        run: |
+          set -euo pipefail
+          # first one might do nothing
+          nix build -L .#wasm32-unknown.ci.ciTestAll5Times --keep-failed
```

Comment:
Interesting, ciTestAll5Times will sometimes not run? Same for wasmTest? Or is this actually a ciTestAll10Times?

Reply:
It's already in the Nix cache, so Nix will just not run it. :shrug: 

We just ensure it's there, then we can run with `--rebuild` which will run it.